api/app/signals/apps/dsl/test.exp

```diff
@@ -0,0 +1,3 @@
+location in stadsdeel.oost
```

Comment:
This file is not referenced anywhere, please remove.

Reply:
deleted